Userland/Applications/Maps/UsersMapWidget.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2023, Bastiaan van der Plaat <bastiaan.v.d.plaat@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include "MapWidget.h"
+
+class UsersMapWidget final : public MapWidget {
+    C_OBJECT(UsersMapWidget);
+
+public:
+    bool show_users() const { return m_show_users; }
+    void set_show_users(bool show_users)
+    {
+        m_show_users = show_users;
+        if (m_show_users) {
+            if (m_users.size() == 0) {
```

Comment:
Technically, the remote list of users could be empty. The neat way to deal with this would be `Optional<Vector<User>>`. But this is a big nitpick :^)